tests/countries/test_finland.py

```diff
@@ -165,6 +166,126 @@ def test_all_saints_day(self):
             "1957-11-01",
         )
 
+    def _test_unofficial_holiday(self, name, month_and_day, year_of_adoption):
+        self.assertHolidayName(
+            name,
+            Finland(categories=UNOFFICIAL),
```

Comment:
Would it be possible to create the unofficial holidays object just once (e.g., `setUpClass()`) and use it w/o function call overhead?

Reply:
Sounds reasonable! I'll update.